src/spurt/workflows/emcf/__init__.py

```diff
@@ -1,10 +1,7 @@
+from ._bulk_offset import get_bulk_offsets
+from ._merge import merge_tiles
+from ._overlap import compute_phasediff_deciles
 from ._settings import GeneralSettings, MergerSettings, SolverSettings, TilerSettings
 from ._solver import EMCFSolver as Solver
-
-__all__ = [
-    "Solver",
-    "GeneralSettings",
-    "MergerSettings",
-    "SolverSettings",
-    "TilerSettings",
-]
+from ._tiling import get_tiles
+from ._unwrap import unwrap_tiles
```

Comment:
Ah sorry let's keep the `__all__` attribute here since we're not using wildcard imports

```suggestion
from ._unwrap import unwrap_tiles

__all__ = [
    "Solver",
    "GeneralSettings",
    "MergerSettings",
    "SolverSettings",
    "TilerSettings",
    "compute_phasediff_deciles",
    "get_bulk_offsets",
    "get_tiles",
    "merge_tiles",
    "unwrap_tiles",
]
```